test/controllers/api/mentoring/requests_controller_test.rb

```diff
@@ -106,6 +107,33 @@ class API::Mentoring::RequestsControllerTest < API::BaseTestCase
     assert_equal user, request.reload.locks.last.locked_by
   end
 
+  test "locked_until should be updated to 30 mins from now" do
```

Comment:
If you wrap this in `freeze_time` then you don't need the `delta` below, and can check for equality instead.